etc/postgres_connect_check/pach_postgres_check.go

```diff
@@ -0,0 +1,36 @@
+// pach_postgres_check.go
+
+package main
+
+import (
+    "context"
+    "fmt"
+    "os"
+    "flag"
+    "time"
+
+    "github.com/jackc/pgx/v5"
+)
+
+func main() {
+
+    connectionString := flag.String("connection_string", "", "postgres database connection string")
```

Comment:
Does this connection string want to be in pgx's format, or in pachyderm's format (see service_env.go/initDBClient; we build it from environment variables)?

It looks like pgbouncer is configured in our setup with $POSTGRESQL_USERNAME, $POSTGRESQL_DATABASE, $POSTGRESQL_HOST, and $POSTGRESQL_PASSWORD and it would be convenient for this tool to take it in that form as well.